app/views/users/show.html.erb

```diff
@@ -1,7 +1,11 @@
 <main>
     <div class="list_item">
         <div class="photo">
-            <%= image_tag (@user.photo) %>
+             <% if @user.photo.present? %>
+                <%= image_tag user.photo %>
+              <% else %>
+                <%= image_tag 'https://www.pngkey.com/png/detail/233-2332677_image-500580-placeholder-transparent.png' %>
```

Comment:
- [ Required ] Please consider fixing the `width` of this image so that it matches the width of the image in the project wireframe. Currently, the image is too big making it overlap with other elements on the page, this makes it difficult for a user to click on the hidden links. Please see the screenshot below and rectify this.

## Your design
![Screenshot from 2023-06-22 01-01-56](https://github.com/ernestmusong/Blog-app/assets/16392046/63487731-01dc-4e0d-9a5c-f9ec68d2d7d7)

## Project wireframe
![](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_root_users.png)